src/content/developers/docs/programming-languages/ruby/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Ethereum for Ruby Developers
+description: Learn how to develop for Ethereum using Ruby-based projects and tooling.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+<div class="featured">Learn how to develop for Ethereum using Ruby-based projects and tooling.</div>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+## Getting started with smart contracts and the Solidity language {#getting-started-with-smart-contracts-and-solidity}
+
+**Take your first steps to integrating Ruby with Ethereum**
+
+Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethereum.org/developers](/developers/).
+
+- [Blockchain Explained](https://kauri.io/article/d55684513211466da7f8cc03987607d5/blockchain-explained)
+- [Understanding Smart Contracts](https://kauri.io/article/e4f66c6079e74a4a9b532148d3158188/ethereum-101-part-5-the-smart-contract)
+- [Write your First Smart Contract](https://kauri.io/article/124b7db1d0cf4f47b414f8b13c9d66e2/remix-ide-your-first-smart-contract)
+- [Learn How to Compile and Deploy Solidity](https://kauri.io/article/973c5f54c4434bb1b0160cff8c695369/understanding-smart-contract-compilation-and-deployment)
+
+## Beginner articles {#beginner-articles}
+
+- [Finally understanding Ethereum accounts](https://dev.to/q9/finally-understanding-ethereum-accounts-1kpe)
+- [How to connect to the Ethereum network using Ruby](https://www.quicknode.com/guides/web3-sdks/how-to-connect-to-the-ethereum-network-using-ruby)
+- [How to generate a new Ethereum address in Ruby](https://www.quicknode.com/guides/web3-sdks/how-to-generate-a-new-ethereum-address-in-ruby)
+
+## Intermediate articles {#intermediate-articles}
+
+- [Blockchain App with Ruby](https://www.nopio.com/blog/blockchain-app-ruby/)
+- [Use the Ruby, connected to Ethereum, to execute the Smart Contract](https://titanwolf.org/Network/Articles/Article?AID=87285822-9b25-49d5-ba2a-7ad95fff7ef9)
+
+## Ruby projects and tools {#ruby-projects-and-tools}
+
+### Active: {#active}
```

Comment:
I know this template was copied from another programming language but we shouldn't have colons at the end of headings.

<img width="314" alt="Screenshot 2022-02-22 at 11 11 54" src="https://user-images.githubusercontent.com/62268199/155121083-8d71ab6e-b0a2-47f1-8dea-250767169cda.png">

```suggestion
### Active {#active}
```